cms/djangoapps/contentstore/rest_api/v2/serializers/home.py

```diff
@@ -0,0 +1,63 @@
+"""
+API Serializers for course home V2 API.
+"""
+from django.conf import settings
+from rest_framework import serializers
+
+from cms.djangoapps.contentstore.utils import get_lms_link_for_item, reverse_course_url
+from cms.djangoapps.contentstore.views.course import _get_rerun_link_for_item
+from openedx.core.lib.api.serializers import CourseKeyField
+
+
+class UnsucceededCourseSerializerV2(serializers.Serializer):
+    """Serializer for unsucceeded course"""
+    display_name = serializers.CharField()
+    course_key = CourseKeyField()
+    org = serializers.CharField()
+    number = serializers.CharField()
+    run = serializers.CharField()
+    is_failed = serializers.BooleanField()
+    is_in_progress = serializers.BooleanField()
+    dismiss_link = serializers.CharField()
+
+
+class CourseCommonSerializerV2(serializers.Serializer):
+    """Serializer for course common fields V2."""
+
+    course_key = CourseKeyField(source='id')
+    display_name = serializers.CharField()
+    lms_link = serializers.SerializerMethodField()
+    cms_link = serializers.SerializerMethodField()
+    number = serializers.CharField()
+    org = serializers.CharField()
+    rerun_link = serializers.SerializerMethodField()
+    run = serializers.CharField(source='id.run')
+    url = serializers.SerializerMethodField()
+    is_active = serializers.SerializerMethodField()
+
+    def get_lms_link(self, obj):
+        """Get LMS link for course."""
+        return get_lms_link_for_item(obj.location)
+
+    def get_cms_link(self, obj):
+        """Get CMS link for course."""
+        return f"//{settings.CMS_BASE}/{reverse_course_url('course_handler', obj.id)}"
```

Comment:
Hi @mariajgrimaldi, in the response the `cms_link` has an additional slash (/)
![image](https://github.com/openedx/edx-platform/assets/64033729/dbeac454-3d73-4201-a410-3da519727516)

Reply:
Thank you! I fixed it. Could you check again?